server/Dockerfile

```diff
@@ -19,7 +19,7 @@ COPY --from=builder /opt /opt
 COPY --from=builder /kadalu /kadalu
 
 RUN apt-get update -yq && \
-    apt-get install -y --no-install-recommends python3 xfsprogs libtirpc3 sqlite3 && \
+    apt-get install -y --no-install-recommends attr python3 xfsprogs libtirpc3 sqlite3 && \
```

Comment:
nit: is this new dependency used in any code?

Reply:
I do not see `getfattr` in `server` pod so Amar is adding this.

We can get away with python xattr but i feel, this is a good addition.